docs/ru/sql-reference/functions/type-conversion-functions.md

```diff
@@ -1074,77 +1074,7 @@ SELECT parseDateTimeBestEffort('10 20:19');
 
 ## parseDateTimeBestEffortUS {#parsedatetimebesteffortUS}
 
-Эта функция похожа на [‘parseDateTimeBestEffort’](#parsedatetimebesteffort), но разница состоит в том, что в она предполагает американский формат даты (`MM/DD/YYYY` etc.) в случае неоднозначности.
-
-**Синтаксис**
-
-``` sql
-parseDateTimeBestEffortUS(time_string [, time_zone])
-```
-
-**Аргументы**
-
--   `time_string` — строка, содержащая дату и время для преобразования. [String](../../sql-reference/data-types/string.md).
--   `time_zone` — часовой пояс. Функция анализирует `time_string` в соответствии с часовым поясом. [String](../../sql-reference/data-types/string.md).
-
-**Поддерживаемые нестандартные форматы**
-
--   Строка, содержащая 9-10 цифр [unix timestamp](https://en.wikipedia.org/wiki/Unix_time).
--   Строка, содержащая дату и время: `YYYYMMDDhhmmss`, `MM/DD/YYYY hh:mm:ss`, `MM-DD-YY hh:mm`, `YYYY-MM-DD hh:mm:ss`, etc.
--   Строка с датой, но без времени: `YYYY`, `YYYYMM`, `YYYY*MM`, `MM/DD/YYYY`, `MM-DD-YY` etc.
--   Строка, содержащая день и время: `DD`, `DD hh`, `DD hh:mm`. В этом случае `YYYY-MM` заменяется на `2000-01`.
--   Строка, содержащая дату и время, а также информацию о часовом поясе: `YYYY-MM-DD hh:mm:ss ±h:mm` и т.д. Например, `2020-12-12 17:36:00 -5:00`.
-
-**Возвращаемое значение**
-
--   `time_string` преобразован в тип данных `DateTime`.
-
-**Примеры**
-
-Запрос:
-
-``` sql
-SELECT parseDateTimeBestEffortUS('09/12/2020 12:12:57')
-AS parseDateTimeBestEffortUS;
-```
-
-Результат:
-
-``` text
-┌─parseDateTimeBestEffortUS─┐
-│     2020-09-12 12:12:57   │
-└─────────────────────────——┘
-```
-
-Запрос:
-
-``` sql
-SELECT parseDateTimeBestEffortUS('09-12-2020 12:12:57')
-AS parseDateTimeBestEffortUS;
-```
-
-Результат:
-
-``` text
-┌─parseDateTimeBestEffortUS─┐
-│     2020-09-12 12:12:57   │
-└─────────────────────────——┘
-```
-
-Запрос:
-
-``` sql
-SELECT parseDateTimeBestEffortUS('09.12.2020 12:12:57')
-AS parseDateTimeBestEffortUS;
-```
-
-Результат:
-
-``` text
-┌─parseDateTimeBestEffortUS─┐
-│     2020-09-12 12:12:57   │
-└─────────────────────────——┘
-```
+Эта функция ведет себя как [parseDateTimeBestEffort](#parsedatetimebesteffort) для форматов даты ISO, например, `YYYY-MM-DD hh:mm:ss`, и других форматов даты, где компоненты месяца и даты могут быть однозначно выделены, например, `YYYYMMDDhhmmss`, `YYYY-MM`, `DD hh`, или `YYYY-MM-DD hh:mm:ss ±h:mm`. Если месяц и компонент даты не могут быть однозначно выделены, например, `MM/DD/YYY`, `MM-DD-YYYY` или `MM-DD-YY`, то вместо `DD/MM/YYY`, `DD-MM-YYYY` или `DD-MM-YY` предпочитается формат даты США. В качестве исключения из последнего, если месяц больше 12 и меньше или равен 31, эта функция возвращается к поведению [parseDateTimeBestEffort](#parsedatetimebesteffort), т.е. `15/08/2020` будет разобрано как `2020-08-15`.
```

Comment:
Translated by Deepl. Native speakers, please let me know if the text sounds awkward.